packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md

```diff
@@ -34,23 +34,23 @@ decide to also use the widget). The list of settings is shown below:
 
 For the sample app use the **Issuer URI** from the custom authorization server
 named “default”.  To find this value, go to **Security > API** in the Admin
-console. Under the **Authorization servers** tab, the default authorization
+Console. Under the **Authorization servers** tab, the default authorization
 server’s Issuer URI is displayed in the list.
 
 An example of the URI is:  `https://dev-foo.okta.com/oauth2/default`.
 
 Note that there are two main types of authorization servers: Org and custom:
 
-1. **Org**: If you want to use the Org authorization server use the following
+* **Org**: If you want to use the Org authorization server use the following
    url: `https://dev-foo.okta.com`.
-1. **Custom**: There are two types of custom authorization servers:
-   1. **default**: The default authorization server is predefined and
+* **Custom**: There are two types of custom authorization servers:
```

Comment:
this isn't necessarily accurate. There is the org auth server and then there are custom auth servers. The FIRST custom auth server is automatically created for them. Then, they can create their own additional custom auth servers. Saying that there are two different types of custom auth servers implies that there is a difference between the custom auth server ("default") that is automatically created for them in their org and the ones that they subsequently create after that. I feel that this wording is misleading.

Reply:
Will improve the clarity of the text in the next PR.